apps/rush-lib/src/utilities/JsonFilePaths.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Feels like this could be made into some form of utility instead of needing to be re-done for each project?

Reply:
Unfortunately, the `require` statement has to be static for Webpack to be able to bundle it.